### PR TITLE
1 new caller workflow templates

### PR DESCRIPTION
**Pull request by victorggonzalez**
added the caller workflow templates from graasp deploy, so they can be accessed directly from the Actions tab of any repository

### CODE REVIEW

File: workflow-templates/cdelivery-s3-apps-caller.properties.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Continuous Delivery Workflow for S3-apps deployments",
+  "description": "Build, test and push a package to Amazon S3 and deploy to CloudFront staging environment.",
```
Comment (pyphilia) [inline workflow-templates/cdelivery-s3-apps-caller.properties.json:3]:
```suggestion
"description": "Build, test and push an app to Amazon S3 and deploy to CloudFront staging environment.",
```

File: workflow-templates/cdeployment-s3-apps-caller.properties.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Continuous Deployment Workflow for S3-apps deployments",
+  "description": "Build, test and push a package to Amazon S3 and deploy to CloudFront production environment.",
```
Comment (pyphilia) [inline workflow-templates/cdeployment-s3-apps-caller.properties.json:3]:
```suggestion
"description": "Build, test and push an app to Amazon S3 and deploy to CloudFront production environment.",
```

File: workflow-templates/cintegration-s3-caller.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      port:
+      show-notifications:
+      stripe-publishable-key:
+      graasp-analyzer-host:
+      google-key:
+      h5p-integration-url:
+      hidden-item-tag-id:
+      port:
+      sentry-dsn:
+      show-notifications:
```
Comment (pyphilia) [inline workflow-templates/cintegration-s3-caller.yml:49]: Can we sort them?

**Review by pyphilia — CHANGES_REQUESTED** [review]
Looks good, but most of them are copy pasted from graasp-deploy, aren't they? Isn't there a possibility to have them sync? Or move everything to `.github`? Because now it means if we want to add an env variable (in app) we need to change all the app workflows, the deploy's ones and these ones so it's sync. I guess we should discuss.

File: workflow-templates/cintegration-s3-caller.yml (hunk 3 above)
Comment (victorggonzalez) [reply workflow-templates/cintegration-s3-caller.yml:49]: done!

**Comment by victorggonzalez** [comment]
Included changes after review. 

As we talked about, the idea is to remove the templates from graasp-deploy and include them here. That way it's easy to add the workflows directly from the workflows tab.
Now if we want to add an env variable we only need to change these.

**Review by pyphilia — APPROVED** [review]

**Pull request merged**